x-pack/plugins/security/server/routes/api/public/roles/get.js

```diff
@@ -11,11 +11,31 @@ import { wrapError } from '../../../../lib/errors';
 export function initGetRolesApi(server, callWithRequest, routePreCheckLicenseFn, application) {
 
   const transformKibanaApplicationsFromEs = (roleApplications) => {
```

Comment:
I see that this is functionally covered in `get.test.js`, what are your thoughts on splitting this out into its own file? I feel like it's getting to the point where it's complex enough to stand on its own.

Reply:
I'm not opposed to splitting it out into it's own file. I don't know if I'd want to test it directly or via the endpoints though... I'm really trying to prevent myself from writing all these white-box tests where we'd have to use jest's module mocking functionality to ensure a function was called, as they generally end up being tautological, but in this situation, it would help us reduce some of the duplication... Lemme think on this.